manifests/dotnet.yml

```diff
@@ -179,6 +179,7 @@ tests/:
         Test_Scanners: v1.28.6
       test_telemetry.py:
         Test_TelemetryMetrics: missing_feature
+        Test_Consistent_Configs: missing_feature
```

Comment:
It looks like this is in the wrong subdirectory. This needs to be moved to the `parametric/` subsection